openghg/analyse/_scenario.py

```diff
@@ -1173,10 +1175,16 @@ def _calc_modelled_obs_HiTRes(
         max_h_back = int(hback.values[-1])
 
         # Define full range of dates to select from the flux input
+        # ____ Footprints time data ____
         date_start = time_array[0]
         date_start_back = date_start - np.timedelta64(max_h_back, "h")
         date_end = time_array[-1] + np.timedelta64(1, "s")
 
+        # ____ Flux time data ____
+        date_start_flux = time_array_flux[0]
```

Comment:
I have a question about how best to define `date_start_flux` - with the current setup the flux data read in will be whatever flux data is available in the object store matching the keywords (not based on input `start_date` and `end_date`) so if e.g. we have flux from e.g. 2010 - 2018 and you were comparing against 2023 - what would this set `date_start_flux` to? Would this grab the 2010 flux data for comparison?